
<div data-component="Callout" data-prop="W3sidHlwZSI6Im1keEF0dHJpYnV0ZSIsIm5hbWUiOiJ2YXJpYW50IiwidmFsdWUiOiJpbXBvcnRhbnQifV0=">
  <p>The end-of-life date for this agent version is July 29, 2019. To update to the latest agent version, see <a href="/docs/agents/manage-apm-agents/installation/update-new-relic-agent">Update the agent</a>. For more information, see <a href="/docs/agents/manage-apm-agents/maintenance/new-relic-agent-plugin-end-life-policy">End-of-life policy</a>.</p>
</div>
<h3>Notes</h3>
<p>This release of the Python agent is a hot fix release to address an issue introduced in version 2.22.0.19 when applying additional function traces from the agent configuration.</p>
<p>The agent can be installed using easy_install/pip/distribute via the <a href="https://pypi.python.org/pypi/newrelic">Python Package Index</a> or can be downloaded directly from our <a href="http://download.newrelic.com/python_agent/release/">download site</a>.</p>
<p>For a list of known issues with the Python agent see our online help article on the <a href="https://newrelic.com/docs/python/status-of-python-agent">status of the Python agent</a>.</p>
<p><strong>Bug fixes:</strong></p>
<ul>
  <li>When using the <code>transaction_tracer.function_trace</code> setting in the agent configuration file to apply additional function traces, the trace was not being applied and a message indicating an instrumentation error would appear in the Python agent log file. The error in itself would not have prevented the agent from starting and running, but no data would be collected for the designated functions as intended.</li>
</ul>
